server/session.go

```diff
@@ -0,0 +1,127 @@
+// SPDX-FileCopyrightText: 2022 ANSSI
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"errors"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/google/uuid"
+	"github.com/sirupsen/logrus"
+)
+
+/*
+	The Session type is an abstraction for the BLE go channel
```

Comment:
"the BLE go channel" is unclear. Can you at least point to a file describing it, or better remind us of the context, eg.: "is an abstraction on top of the Go channel used to pass binary data to the goroutine reading and writing on the BLE characteristic. It takes care of automatically encoding and encrypting Go objects before sending them on the channel, and the other way around for received messages."